src/json_parse.rs

```diff
@@ -1,15 +1,15 @@
 use crate::db::{BotDbError, Repo};
-use crate::open_weather_map::weather::City;
-use serde_json::Value;
+use crate::open_weather_map::weather::CityDeserialize;
 use std::fs;
 
 pub async fn read_json_cities() -> Result<(), BotDbError> {
```

Comment:
I think the name of the module and the function should be changed. The function populates cities

Reply:
Done in [8e9feff](https://github.com/pxp9/weather_bot_rust/pull/17/commits/8e9feff9b546089bf98bdedf9ba69874ff2cf740)